client/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+	"env": {
+		"es6": true,
+		"browser": true,
+		"jest": true
+	},
+	"extends": ["airbnb-typescript", "prettier"],
+	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
+		"project": "./tsconfig.json"
+	},
+	"plugins": ["prettier"],
+	"rules": {
+		"prettier/prettier": "error",
+		"indent": "off",
+		"@typescript-eslint/indent": "off",
+		"react/jsx-indent": "off",
+		"react/jsx-indent-props": "off"
```

Comment:
Eslint indentation rules were clashing with prettier so I turned off the indentation rules in order to follow prettier settings.